package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "strip-pragma-loader",
+  "version": "0.0.1",
+  "description": "A webpack loader to strip out code blocks between RequireJS pragma style comments.",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:AnalyticalGraphicsInc/strip-pragma-loader.git"
+  },
+  "bugs": {
+    "url": "https://github.com/AnalyticalGraphicsInc/strip-pragma-loader/issues"
+  },
+  "keywords": [
+    "webpack",
+    "strip",
+    "comments",
+    "require",
+    "requirejs",
+    "pragmas",
+    "debug",
+    "block"
+  ],
+  "author": {
+    "name": "Analytical Graphics, Inc.",
+    "url": "http://www.agi.com"
+  },
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/AnalyticalGraphicsInc/strip-pragma-loader",
+  "peerDependencies": {
+    "webpack": "^2"
```

Comment:
We normally specify the full version (`^2.0.0` in thie case.)  Is there a reason for now doing that here?

Reply:
Just how I saw it noted in other loader packages, I'll go ahead and specify the full version.